test/index.spec.js

```diff
@@ -44,6 +44,12 @@ function createOptions () {
   }
 }
 
+function createRepoOptions () {
+  return {
+
+  }
+}
```

Comment:
Why is this empty? Options are required but can be an empty object? That seems silly.

Reply:
The implementation is a sinon stub so it doesn't do any verification, we just validate that we were passed *something* in the test.